gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfigurationTest.java

```diff
@@ -78,11 +78,12 @@
           put("fs.gs.encryption.key", null);
           put("fs.gs.encryption.key.hash", null);
           put("fs.gs.glob.algorithm", GlobAlgorithm.CONCURRENT);
-          put("fs.gs.grpc.checksums.enable", false);
           put("fs.gs.grpc.enable", false);
+          put("fs.gs.grpc.checksums.enable", false);
+          put("fs.gs.grpc.server.address", null);
```

Comment:
Please sort alphabetically

Reply:
Oh that was intentional. I just presumed that it's a mistake. Done!